base/sparse/cholmod.jl

```diff
@@ -418,6 +418,13 @@ function copy_dense{Tv<:VTypes}(A::Dense{Tv})
     d
 end
 
+function sort!{Tv<:VTypes}(S::Sparse{Tv})
```

Comment:
I have repeatedly had a really hard time debugging code in this file because of the name shadowing. I don't think it's a good convention. At least prefix `cholmod_` or something.